src/realm/object-store/c_api/realm.cpp

```diff
@@ -57,6 +57,15 @@ RLM_API realm_t* realm_open(const realm_config_t* config)
     });
 }
 
+RLM_API bool realm_delete_files(const char* realm_file_path)
```

Comment:
Shouldn't this return an out bool parameter indicating whether an error has occurred? Otherwise there's no way to differentiate between an error occurring and the files not getting deleted.

Reply:
What are the error scenarios? Only thing that I can come up with where deleting the file doesn't delete anything is if the file is not present in the first place. But maybe that is also an error and then I guess the doc should just be more explicit on this? Anyway, if we need both an error and a deletion indicator, then I think we should follow the pattern from other methods, and indicate the error with the return value and the deletion indicator with the output parameter.

Reply:
Two months away from the C API and I already forgot some fundamental principles 😅
I added the `did_delete_realm` out parameter and fixed the return value to be the error indicator.